### ISSUE TITLE
Switching to CEF

### ISSUE THREAD

**Issue by Nesh108** [gree/unity-webview#217]
Hey @KojiNakamaru,

Seeing some problems, incomplete platform support (no Windows and no Linux) and somewhat poor performance, I was wondering if we should start looking into CEF as a definitive answer to most (if not all) of them?

I know you said you might not have time for that but I was wondering what you think would be the steps to make the full switch. Having some sort of checklist, would make it easier for you and other people to work on this task, which I think is very important to make this project be even more useful.

Let me know :)

**Comment by KojiNakamaru** [comment]
As I said before,
https://github.com/gree/unity-webview/issues/122#issuecomment-311846815
if you provide any experimental implementation, it should be very useful, I guess not only for me but others. Could you please start to work on it if you really need such functions and have time? As I said, I don't have time for now to investigate further details.

**Comment by Nesh108** [comment]
Yeah, I know but I also would like to know what kind of stuff did you have to do while making this project.

I would be interested in helping out but I never worked on low-level browser stuff, so some sort of guidance would certainly help loads, especially since we have to make it work with your code 😄

**Comment by KojiNakamaru** [comment]
Please start to read all code in this plugin. Once you understand the structure of details, then you can start to learn how CEF works and start to investigate how CEF can be integrated into this plugin.
